genomedata/_load_data.py

```diff
@@ -175,8 +163,7 @@ def parse_args(args):
 
 def main(argv=sys.argv[1:]):
     args = parse_args(argv)
-    load_data(args.gdarchive, args.trackname, args.datafile,
-              args.maskfile, verbose=args.verbose, chunk_size=args.chunk_size)
+    load_data_from_stdin(args.gdarchive, args.trackname, args.verbose)
```

Comment:
It looks like a bunch of the other functionality like maskfiles are no longer accessible?

Reply:
This is admittedly confusing because the current `_load_data.py` was and is doing double duty.
1. It is solely the source of python functions for loading data for the convenience wrapper `genomedata-load` and as a python API of sorts.
2. It had an old python interface that was never used. It was resurrected in this case as the interface to the c-extension.

The current python `load_data` function is used, it's just used through `genomedata-load` which uses the maskfiles et. al. But as mentioned before the `chunk_size` option is never used oddly enough as well.

Reply:
So there was no entry point that used the old `_load_data.parse_args()`? It was essentially dead code?

Reply:
Yes. Presumably, it was code accidentally left in after the initial transition from Python to the C binary. The only place it was used was for generating documentation. However, the interfaces were technically different so it wasn't even accurate to begin with.